src/index.ts

```diff
@@ -4,8 +4,9 @@ import yargs from 'yargs'
 
 import { createZip } from './zipfile'
 import { postToP4D } from './p4d'
+import { Config } from './config'
 
-async function upload (gameId, buildDir, filename, name, notes, makePublic) {
+async function upload (gameId: string, buildDir: string, filename: string, name: string, notes: string|undefined, makePublic: boolean): Promise<void> {
```

Comment:
```suggestion
async function upload (gameId: string, buildDir: string, filename: string, name: string, notes?: string, makePublic: boolean): Promise<void> {
```

Unless it doesn't work because the parameters after aren't optional? In which case

```suggestion
async function upload (gameId: string, buildDir: string, filename: string, name: string, notes?: string, makePublic = false): Promise<void> {
```

Reply:
No that doesn't work.